src/DankCord/DankCord.py

```diff
@@ -300,7 +299,7 @@ def _post_handling(
             self.logger.error("Did not receive nonce in time.")
             return None
 
-        if message_id != "":
+        if message_id:
```

Comment:
Maybe it would've been better to remove white spaces and check, since even if the string had a single space and nothing else, it'd return True.